torcharrow/test/test_numerical_column.py

```diff
@@ -423,6 +423,12 @@ def base_test_na_handling(self):
         c = c.append([2])
         self.assertEqual(set(c.drop_duplicates()), {None, 2, 17.0})
 
+    def base_test_fill_null_type_promotion_rules(self):
```

Comment:
You need to add entry in https://github.com/facebookresearch/torcharrow/blob/main/torcharrow/test/test_numerical_column_cpu.py#L12 , otherwise, the test won't run